## 59% of end-end tests have passed
```
Total executed tests: 51
Failed tests: 21
Expected to fail tests: 0
Passed tests: 30
```
```
IDs of failed tests: 702840,727231,702894,703086,727230,702958,702844,704613,702841,702839,702838,704615,702859,702869,702957,702843,727229,739307,703503,702777,703194 
```
<h3>Failed tests (21)</h3><details><summary>Click to expand</summary><br/><li><a href="https://ci.status.im/job/status-mobile/job/e2e/job/status-app-prs-rerun/parambuild/?BRANCH_NAME=chore/remove_node_config&PR_ID=20949&APK_NAME=20949.apk&TR_CASE_IDS=702840,727231,702894,703086,727230,702958,702844,704613,702841,702839,702838,704615,702859,702869,702957,702843,727229,739307,703503,702777,703194">Rerun failed tests</a></li><br/><h4>Class TestCommunityOneDeviceMerged:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3640390">test_community_undo_delete_message</a>, id: 702869 </b></td></tr><tr><td><p><blockquote><div>Device 1: Find `Text` by `xpath`: `//android.view.ViewGroup[@content-desc='chat-item']//android.widget.TextView[contains(@text,'message to delete and undo')]` </div><div>Device 1: Long press on `Text` </div></blockquote></p>

```
critical/chats/test_public_chat_browsing.py:74: in test_community_undo_delete_message
    self.channel.delete_message_in_chat(message_to_delete)
../views/chat_view.py:1028: in delete_message_in_chat
    self.chat_element_by_text(message).message_body.long_press_element()
../views/base_element.py:324: in long_press_element
    action.long_press(element).release().perform()
../../../../status-app-prs-rerun@tmp/venv/lib/python3.10/site-packages/appium/webdriver/common/touch_action.py:174: in perform
    self._driver.execute(Command.TOUCH_ACTION, params)
../../../../status-app-prs-rerun@tmp/venv/lib/python3.10/site-packages/selenium/webdriver/remote/webdriver.py:345: in execute
    self.error_handler.check_response(response)
../../../../status-app-prs-rerun@tmp/venv/lib/python3.10/site-packages/appium/webdriver/errorhandler.py:122: in check_response
    raise exception_class(msg=message, stacktrace=format_stacktrace(stacktrace))
 The element 'By.xpath: //android.view.ViewGroup[@content-desc='chat-item']//android.widget.TextView[contains(@text,'message to delete and undo')]' is not linked to the same object in DOM anymore; For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#stale-element-reference-exception
E   Stacktrace:
E   io.appium.uiautomator2.common.exceptions.StaleElementReferenceException: The element 'By.xpath: //android.view.ViewGroup[@content-desc='chat-item']//android.widget.TextView[contains(@text,'message to delete and undo')]' is not linked to the same object in DOM anymore
E   	at io.appium.uiautomator2.model.ElementsCache.restore(ElementsCache.java:122)
E   	at io.appium.uiautomator2.model.ElementsCache.get(ElementsCache.java:153)
E   	at io.appium.uiautomator2.handler.Location.safeHandle(Location.java:23)
E   	at io.appium.uiautomator2.handler.request.SafeRequestHandler.handle(SafeRequestHandler.java:59)
E   	at io.appium.uiautomator2.server.AppiumServlet.handleRequest(AppiumServlet.java:277)
E   	at io.appium.uiautomator2.server.AppiumServlet.handleHttpRequest(AppiumServlet.java:271)
E   	at io.appium.uiautomator2.http.ServerHandler.channelRead(ServerHandler.java:68)
E   	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:366)
E   	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:352)
E   	at io.netty.channel.AbstractChannelHandlerContext.fireChannelRead(AbstractChannelHandlerContext.java:345)
E   	at io.netty.handler.codec.MessageToMessageDecoder.channelRead(MessageToMessageDecoder.java:102)
E   	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:366)
E   	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:352)
E   	at io.netty.channel.AbstractChannelHandlerContext.fireChannelRead(AbstractChannelHandlerContext.java:345)
E   	at io.netty.channel.CombinedChannelDuplexHandler$DelegatingChannelHandlerContext.fireChannelRead(CombinedChannelDuplexHandler.java:435)
E   	at io.netty.handler.codec.ByteToMessageDecoder.fireChannelRead(ByteToMessageDecoder.java:293)
E   	at io.netty.handler.codec.ByteToMessageDecoder.channelRead(ByteToMessageDecoder.java:267)
E   	at io.netty.channel.CombinedChannelDuplexHandler.channelRead(CombinedChannelDuplexHandler.java:250)
E   	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:366)
E   	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:352)
E   	at io.netty.channel.AbstractChannelHandlerContext.fireChannelRead(AbstractChannelHandlerContext.java:345)
E   	at io.netty.handler.timeout.IdleStateHandler.channelRead(IdleStateHandler.java:266)
E   	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:366)
E   	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:352)
E   	at io.netty.channel.AbstractChannelHandlerContext.fireChannelRead(AbstractChannelHandlerContext.java:345)
E   	at io.netty.channel.DefaultChannelPipeline$HeadContext.channelRead(DefaultChannelPipeline.java:1294)
E   	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:366)
E   	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:352)
E   	at io.netty.channel.DefaultChannelPipeline.fireChannelRead(DefaultChannelPipeline.java:911)
E   	at io.netty.channel.nio.AbstractNioByteChannel$NioByteUnsafe.read(AbstractNioByteChannel.java:131)
E   	at io.netty.channel.nio.NioEventLoop.processSelectedKey(NioEventLoop.java:611)
E   	at io.netty.channel.nio.NioEventLoop.processSelectedKeysOptimized(NioEventLoop.java:552)
E   	at io.netty.channel.nio.NioEventLoop.processSelectedKeys(NioEventLoop.java:466)
E   	at io.netty.channel.nio.NioEventLoop.run(NioEventLoop.java:438)
E   	at io.netty.util.concurrent.SingleThreadEventExecutor$2.run(SingleThreadEventExecutor.java:140)
E   	at io.netty.util.concurrent.DefaultThreadFactory$DefaultRunnableDecorator.run(DefaultThreadFactory.java:144)
E   	at java.lang.Thread.run(Thread.java:1012)
```

<br/><br/><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/81643fc90b3d46b48c5faece56489f96?auth=36f6635e58cfbbfa8f01bbbede4786c1#104">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3640405">test_community_discovery</a>, id: 703503 </b></td></tr><tr><td><p><blockquote><div>Device 1: Find `Button` by `accessibility id`: `communities-home-discover-card` </div><div>Device 1: Tap on found: Button </div></blockquote></p>

```
critical/chats/test_public_chat_browsing.py:262: in test_community_discovery
    self.home.community_card_item.wait_for_elements(seconds=120)
../views/base_element.py:138: in wait_for_elements
    raise TimeoutException(
 Device 1:  BaseElement by accessibility id:`community-card-item` is not found on the screen after wait_for_elements
```

<br/><br/><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/81643fc90b3d46b48c5faece56489f96?auth=36f6635e58cfbbfa8f01bbbede4786c1#346">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestActivityMultipleDevicePRTwo:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3640395">test_activity_center_admin_notification_accept_swipe</a>, id: 702958 </b></td></tr><tr><td>

```
Test setup failed: activity_center/test_activity_center.py:414: in prepare_devices
    self.community_2.join_community()
../views/chat_view.py:419: in join_community
    ChatView(self.driver).chat_element_by_text("https://status.app/c/").click_on_link_inside_message_body()
../views/chat_view.py:175: in click_on_link_inside_message_body
    self.message_body.wait_for_visibility_of_element(30)
../views/base_element.py:147: in wait_for_visibility_of_element
    raise TimeoutException(
 Device 2: Text by xpath:`//android.view.ViewGroup[@content-desc='chat-item']//android.widget.TextView[contains(@text,'https://status.app/c/')]` is not found on the screen after wait_for_visibility_of_element
```

<br/><br/></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3640394">test_activity_center_mentions</a>, id: 702957 </b></td></tr><tr><td><p><blockquote><div>Device 2: Joining community </div><div>Device 2: Looking for a message by text: https://status.app/c/ </div></blockquote></p>

```
Test setup failed: activity_center/test_activity_center.py:414: in prepare_devices
    self.community_2.join_community()
../views/chat_view.py:419: in join_community
    ChatView(self.driver).chat_element_by_text("https://status.app/c/").click_on_link_inside_message_body()
../views/chat_view.py:175: in click_on_link_inside_message_body
    self.message_body.wait_for_visibility_of_element(30)
../views/base_element.py:147: in wait_for_visibility_of_element
    raise TimeoutException(
 Device 2: Text by xpath:`//android.view.ViewGroup[@content-desc='chat-item']//android.widget.TextView[contains(@text,'https://status.app/c/')]` is not found on the screen after wait_for_visibility_of_element
```

<br/><br/><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/bcb042dc2f0d4773be97e7acde66c92b?auth=2a351dcb5df725095bacb20654ed1ea7">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/bae0e61a6ed54684a6cf81bc54a9c883?auth=e0b6c3870841612eeaf37547f8dc555d">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestWalletMultipleDevice:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3640410">test_wallet_send_asset_from_drawer</a>, id: 727230 </b></td></tr><tr><td><p><blockquote><div># STEP: Getting ETH amount in the wallet of the sender before transaction</div><div>Device 1: Find `Button` by `xpath`: `//android.view.ViewGroup[contains(@content-desc,'Account 1')]` </div></blockquote></p>

```
critical/test_wallet.py:164: in test_wallet_send_asset_from_drawer
    sender_balance, receiver_balance, eth_amount_sender, eth_amount_receiver = self._get_balances_before_tx()
critical/test_wallet.py:43: in _get_balances_before_tx
    self.wallet_1.get_account_element().click()
../views/base_element.py:90: in click
    element = self.find_element()
../views/base_element.py:79: in find_element
    raise NoSuchElementException(
 Device 1: Button by xpath: `//android.view.ViewGroup[contains(@content-desc,'Account 1')]` is not found on the screen; For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception
```

<br/><br/></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3640409">test_wallet_send_eth</a>, id: 727229 </b></td></tr><tr><td><p><blockquote><div>Device 1: Swiping right on element SlideButton </div><div>Device 1: Find `SlideButton` by `xpath`: `//*[@resource-id='slide-button-track']` </div></blockquote></p>

```
critical/test_wallet.py:139: in test_wallet_send_eth
    self.wallet_1.send_asset(address='arb1:' + self.receiver['wallet_address'],
../views/wallet_view.py:148: in send_asset
    self.confirm_transaction()
../views/wallet_view.py:131: in confirm_transaction
    self.slide_and_confirm_with_password()
../views/wallet_view.py:120: in slide_and_confirm_with_password
    self.slide_button_track.slide()
../views/base_view.py:257: in slide
    self.swipe_right_on_element(width_percentage=1.3, start_x=100)
../views/base_element.py:308: in swipe_right_on_element
    location, size = self.get_element_coordinates()
../views/base_element.py:294: in get_element_coordinates
    element = self.find_element()
../views/base_element.py:79: in find_element
    raise NoSuchElementException(
 Device 1: SlideButton by xpath: `//*[@resource-id='slide-button-track']` is not found on the screen; For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception
```

<br/><br/></td></tr></tbody></table><h4>Class TestDeepLinksOneDevice:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3640407">test_links_open_universal_links_from_chat</a>, id: 704613 </b></td></tr><tr><td><p><blockquote><div>Device 1: Tap on found: SendMessageButton </div><div>Device 1: Looking for a message by text: https://status.app/u#zQ3shVVxZMwLVEQvuu1KF6h4D2mzVyCC4F4mHLZm5dz5XU1aa </div></blockquote></p>

```
critical/test_deep_and_universal_links.py:41: in test_links_open_universal_links_from_chat
    self.channel.chat_element_by_text(url).click_on_link_inside_message_body()
../views/chat_view.py:175: in click_on_link_inside_message_body
    self.message_body.wait_for_visibility_of_element(30)
../views/base_element.py:147: in wait_for_visibility_of_element
    raise TimeoutException(
 Device 1: Text by xpath:`//android.view.ViewGroup[@content-desc='chat-item']//android.widget.TextView[contains(@text,'https://status.app/u#zQ3shVVxZMwLVEQvuu1KF6h4D2mzVyCC4F4mHLZm5dz5XU1aa')]` is not found on the screen after wait_for_visibility_of_element
```

<br/><br/><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/48055c034b0240fe8e693c2ea6eeaa95?auth=3ce5de0bbc71c7337daff64a1c6d8043#59">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3640412">test_deep_links_communities</a>, id: 739307 </b></td></tr><tr><td><p><blockquote><div>Device 1: Tap on found: Button </div><div>Device 1: Find `Button` by `accessibility id`: `share-community` </div></blockquote></p>

```
critical/test_deep_and_universal_links.py:85: in test_deep_links_communities
    closed_community_url = self.community_view.copy_community_link()
../views/chat_view.py:458: in copy_community_link
    self.share_community_button.click()
../views/base_element.py:90: in click
    element = self.find_element()
../views/base_element.py:79: in find_element
    raise NoSuchElementException(
 Device 1: Button by accessibility id: `share-community` is not found on the screen; For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception
```

<br/><br/><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/48055c034b0240fe8e693c2ea6eeaa95?auth=3ce5de0bbc71c7337daff64a1c6d8043#169">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestActivityCenterContactRequestMultipleDevicePR:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3640369">test_add_contact_field_validation</a>, id: 702777 </b></td></tr><tr><td><p><blockquote><div>Device 1: Find `Button` by `xpath`: `//*[@text="Paste"]` </div><div>Device 1: Tap on found: Button </div></blockquote></p>

```
activity_center/test_activity_center.py:237: in test_add_contact_field_validation
    self.errors.verify_no_errors()
base_test_case.py:191: in verify_no_errors
    pytest.fail('\n '.join([self.errors.pop(0) for _ in range(len(self.errors))]))
 Error message is not shown for value "zQ3shvk8ZqvfQ8HD5foimd843bqJwzgmD85c5RN7jgC3RR2Y"
E    Error message is not shown for value "0x512543419b894307EA979C8d24219c279Be2315B"
E    Error message is not shown for value "cs3:X1cQV8s:Cce:fw7WH45a4waCCrTmNHbtW1zGhz9WezmryTjZLFdgLnzs:3tGyLJiSXNsKUuHeRNRL85yd56Xb5gerEEKmaRzxpkH9"
```

<br/><br/><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/a09319a961834f8891dea39d22a77483?auth=2b2625d97e2ebdbdfe6c881eb48a0e36#138">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/596798caa9f9468fb11e9d949bb0626f?auth=4e96478e39578d4e2cf7d6ce9c29003e#204">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestCommunityMultipleDeviceMerged:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3640380">test_community_emoji_send_copy_paste_reply</a>, id: 702840 </b></td></tr><tr><td>

```
Test setup failed: critical/chats/test_public_chat_browsing.py:346: in prepare_devices
    self.chat_2.send_message(self.text_message)
../views/chat_view.py:981: in send_message
    self.chat_message_input.wait_for_element(wait_chat_input_sec)
../views/base_element.py:129: in wait_for_element
    raise TimeoutException(
 Device `2`: `ChatMessageInput` by` accessibility id`: `chat-message-input` is not found on the screen after wait_for_element
```

<br/><br/></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3640391">test_community_contact_block_unblock_offline</a>, id: 702894 </b></td></tr><tr><td>

```
Test setup failed: critical/chats/test_public_chat_browsing.py:346: in prepare_devices
    self.chat_2.send_message(self.text_message)
../views/chat_view.py:981: in send_message
    self.chat_message_input.wait_for_element(wait_chat_input_sec)
../views/base_element.py:129: in wait_for_element
    raise TimeoutException(
 Device `2`: `ChatMessageInput` by` accessibility id`: `chat-message-input` is not found on the screen after wait_for_element
```

<br/><br/></td></tr><tr><td><b>3. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3640396">test_community_mark_all_messages_as_read</a>, id: 703086 </b></td></tr><tr><td>

```
Test setup failed: critical/chats/test_public_chat_browsing.py:346: in prepare_devices
    self.chat_2.send_message(self.text_message)
../views/chat_view.py:981: in send_message
    self.chat_message_input.wait_for_element(wait_chat_input_sec)
../views/base_element.py:129: in wait_for_element
    raise TimeoutException(
 Device `2`: `ChatMessageInput` by` accessibility id`: `chat-message-input` is not found on the screen after wait_for_element
```

<br/><br/></td></tr><tr><td><b>4. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3640383">test_community_links_with_previews_github_youtube_twitter_gif_send_enable</a>, id: 702844 </b></td></tr><tr><td>

```
Test setup failed: critical/chats/test_public_chat_browsing.py:346: in prepare_devices
    self.chat_2.send_message(self.text_message)
../views/chat_view.py:981: in send_message
    self.chat_message_input.wait_for_element(wait_chat_input_sec)
../views/base_element.py:129: in wait_for_element
    raise TimeoutException(
 Device `2`: `ChatMessageInput` by` accessibility id`: `chat-message-input` is not found on the screen after wait_for_element
```

<br/><br/></td></tr><tr><td><b>5. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3640381">test_community_unread_messages_badge</a>, id: 702841 </b></td></tr><tr><td>

```
Test setup failed: critical/chats/test_public_chat_browsing.py:346: in prepare_devices
    self.chat_2.send_message(self.text_message)
../views/chat_view.py:981: in send_message
    self.chat_message_input.wait_for_element(wait_chat_input_sec)
../views/base_element.py:129: in wait_for_element
    raise TimeoutException(
 Device `2`: `ChatMessageInput` by` accessibility id`: `chat-message-input` is not found on the screen after wait_for_element
```

<br/><br/></td></tr><tr><td><b>6. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3640379">test_community_message_delete</a>, id: 702839 </b></td></tr><tr><td>

```
Test setup failed: critical/chats/test_public_chat_browsing.py:346: in prepare_devices
    self.chat_2.send_message(self.text_message)
../views/chat_view.py:981: in send_message
    self.chat_message_input.wait_for_element(wait_chat_input_sec)
../views/base_element.py:129: in wait_for_element
    raise TimeoutException(
 Device `2`: `ChatMessageInput` by` accessibility id`: `chat-message-input` is not found on the screen after wait_for_element
```

<br/><br/></td></tr><tr><td><b>7. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3640378">test_community_message_send_check_timestamps_sender_username</a>, id: 702838 </b></td></tr><tr><td><p><blockquote><div># STEP: Send message to contact (need for blocking contact) test</div><div>Device 2: Sending message 'hello' </div></blockquote></p>

```
Test setup failed: critical/chats/test_public_chat_browsing.py:346: in prepare_devices
    self.chat_2.send_message(self.text_message)
../views/chat_view.py:981: in send_message
    self.chat_message_input.wait_for_element(wait_chat_input_sec)
../views/base_element.py:129: in wait_for_element
    raise TimeoutException(
 Device `2`: `ChatMessageInput` by` accessibility id`: `chat-message-input` is not found on the screen after wait_for_element
```

<br/><br/><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/4aecfd6c8c3a4cb69b75b6013c7ce4b9?auth=94309405a30ae1e0e49b4aada19f1016">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/778670689e0142f19ab4b3a587724aaf?auth=3a0fbadc75b20c09918ad20dcea70a33">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>8. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3640408">test_community_edit_delete_message_when_offline</a>, id: 704615 </b></td></tr><tr><td>

```
Test setup failed: critical/chats/test_public_chat_browsing.py:346: in prepare_devices
    self.chat_2.send_message(self.text_message)
../views/chat_view.py:981: in send_message
    self.chat_message_input.wait_for_element(wait_chat_input_sec)
../views/base_element.py:129: in wait_for_element
    raise TimeoutException(
 Device `2`: `ChatMessageInput` by` accessibility id`: `chat-message-input` is not found on the screen after wait_for_element
```

<br/><br/></td></tr><tr><td><b>9. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3640389">test_community_one_image_send_reply</a>, id: 702859 </b></td></tr><tr><td>

```
Test setup failed: critical/chats/test_public_chat_browsing.py:346: in prepare_devices
    self.chat_2.send_message(self.text_message)
../views/chat_view.py:981: in send_message
    self.chat_message_input.wait_for_element(wait_chat_input_sec)
../views/base_element.py:129: in wait_for_element
    raise TimeoutException(
 Device `2`: `ChatMessageInput` by` accessibility id`: `chat-message-input` is not found on the screen after wait_for_element
```

<br/><br/></td></tr><tr><td><b>10. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3640382">test_community_message_edit</a>, id: 702843 </b></td></tr><tr><td>

```
Test setup failed: critical/chats/test_public_chat_browsing.py:346: in prepare_devices
    self.chat_2.send_message(self.text_message)
../views/chat_view.py:981: in send_message
    self.chat_message_input.wait_for_element(wait_chat_input_sec)
../views/base_element.py:129: in wait_for_element
    raise TimeoutException(
 Device `2`: `ChatMessageInput` by` accessibility id`: `chat-message-input` is not found on the screen after wait_for_element
```

<br/><br/></td></tr><tr><td><b>11. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3640398">test_community_several_images_send_reply</a>, id: 703194 </b></td></tr><tr><td>

```
Test setup failed: critical/chats/test_public_chat_browsing.py:346: in prepare_devices
    self.chat_2.send_message(self.text_message)
../views/chat_view.py:981: in send_message
    self.chat_message_input.wait_for_element(wait_chat_input_sec)
../views/base_element.py:129: in wait_for_element
    raise TimeoutException(
 Device `2`: `ChatMessageInput` by` accessibility id`: `chat-message-input` is not found on the screen after wait_for_element
```

<br/><br/></td></tr></tbody></table><h4>Class TestWalletOneDevice:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3640411">test_wallet_add_remove_regular_account</a>, id: 727231 </b></td></tr><tr><td><p><blockquote><div>Device 1: Find `Button` by `accessibility id`: `account-emoji` </div><div>Device 1: Find `Button` by `accessibility id`: `share-account` </div></blockquote></p>

```
critical/test_wallet.py:215: in test_wallet_add_remove_regular_account
    self.wallet_view.share_address_button.click()
../views/base_element.py:90: in click
    element = self.find_element()
../views/base_element.py:79: in find_element
    raise NoSuchElementException(
 Device 1: Button by accessibility id: `share-account` is not found on the screen; For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception
```

<br/><br/><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/b240d0f45e434cd6a58fad407672feac?auth=f88d79ae823a6f6d0dd4ab34eb819128#37">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table></details><h3>Passed tests (30)</h3><details><summary>Click to expand</summary><br/><br/><h4>Class TestCommunityOneDeviceMerged:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3640366">test_community_copy_and_paste_message_in_chat_input</a>, id: 702742 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/81643fc90b3d46b48c5faece56489f96?auth=36f6635e58cfbbfa8f01bbbede4786c1#75">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3640385">test_community_navigate_to_channel_when_relaunch</a>, id: 702846 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/81643fc90b3d46b48c5faece56489f96?auth=36f6635e58cfbbfa8f01bbbede4786c1#51">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>3. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3640397">test_restore_multiaccount_with_waku_backup_remove_switch</a>, id: 703133 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/81643fc90b3d46b48c5faece56489f96?auth=36f6635e58cfbbfa8f01bbbede4786c1#221">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>4. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3640401">test_community_mute_community_and_channel</a>, id: 703382 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/81643fc90b3d46b48c5faece56489f96?auth=36f6635e58cfbbfa8f01bbbede4786c1#116">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestOneToOneChatMultipleSharedDevicesNewUi:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3640388">test_1_1_chat_edit_message</a>, id: 702855 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/7222c8519ff44f0e9f4cf8223d068c4d?auth=eab1c095cd8306ebd1c116e61e8e61a9#462">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/e2c0fe2887e846749a560a4a5635b2bb?auth=da2ac96d5261e6d0f89750fcc2f88ab4#298">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3640362">test_1_1_chat_message_reaction</a>, id: 702730 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/7222c8519ff44f0e9f4cf8223d068c4d?auth=eab1c095cd8306ebd1c116e61e8e61a9#99">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/e2c0fe2887e846749a560a4a5635b2bb?auth=da2ac96d5261e6d0f89750fcc2f88ab4#88">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>3. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3640367">test_1_1_chat_non_latin_messages_stack_update_profile_photo</a>, id: 702745 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/7222c8519ff44f0e9f4cf8223d068c4d?auth=eab1c095cd8306ebd1c116e61e8e61a9#355">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/e2c0fe2887e846749a560a4a5635b2bb?auth=da2ac96d5261e6d0f89750fcc2f88ab4#215">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>4. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3640363">test_1_1_chat_pin_messages</a>, id: 702731 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/7222c8519ff44f0e9f4cf8223d068c4d?auth=eab1c095cd8306ebd1c116e61e8e61a9#217">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/e2c0fe2887e846749a560a4a5635b2bb?auth=da2ac96d5261e6d0f89750fcc2f88ab4#170">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>5. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3640365">test_1_1_chat_text_message_delete_push_disappear</a>, id: 702733 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/7222c8519ff44f0e9f4cf8223d068c4d?auth=eab1c095cd8306ebd1c116e61e8e61a9#554">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/e2c0fe2887e846749a560a4a5635b2bb?auth=da2ac96d5261e6d0f89750fcc2f88ab4#408">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>6. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3640377">test_1_1_chat_push_emoji</a>, id: 702813 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/7222c8519ff44f0e9f4cf8223d068c4d?auth=eab1c095cd8306ebd1c116e61e8e61a9#413">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/e2c0fe2887e846749a560a4a5635b2bb?auth=da2ac96d5261e6d0f89750fcc2f88ab4#265">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>7. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3640370">test_1_1_chat_emoji_send_reply_and_open_link</a>, id: 702782 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/7222c8519ff44f0e9f4cf8223d068c4d?auth=eab1c095cd8306ebd1c116e61e8e61a9#151">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/e2c0fe2887e846749a560a4a5635b2bb?auth=da2ac96d5261e6d0f89750fcc2f88ab4#125">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>8. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3640402">test_1_1_chat_send_image_save_and_share</a>, id: 703391 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/7222c8519ff44f0e9f4cf8223d068c4d?auth=eab1c095cd8306ebd1c116e61e8e61a9#493">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/e2c0fe2887e846749a560a4a5635b2bb?auth=da2ac96d5261e6d0f89750fcc2f88ab4#348">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestOneToOneChatMultipleSharedDevicesNewUiTwo:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3640404">test_1_1_chat_mute_chat</a>, id: 703496 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/bf7d298270cd4365b89eca6c28e8de83?auth=ed8e582e938234b1e72847ee7014b63b#109">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/ce0f91da720e4280a5a33ccad435db64?auth=e4710f4277f0748ba41a6f5da80176f2#143">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3640371">test_1_1_chat_is_shown_message_sent_delivered_from_offline</a>, id: 702783 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/bf7d298270cd4365b89eca6c28e8de83?auth=ed8e582e938234b1e72847ee7014b63b#99">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/ce0f91da720e4280a5a33ccad435db64?auth=e4710f4277f0748ba41a6f5da80176f2#88">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>3. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3640372">test_1_1_chat_delete_via_long_press_relogin</a>, id: 702784 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/bf7d298270cd4365b89eca6c28e8de83?auth=ed8e582e938234b1e72847ee7014b63b#175">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/ce0f91da720e4280a5a33ccad435db64?auth=e4710f4277f0748ba41a6f5da80176f2#156">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestDeepLinksOneDevice:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3640368">test_links_deep_links_profile</a>, id: 702775 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/48055c034b0240fe8e693c2ea6eeaa95?auth=3ce5de0bbc71c7337daff64a1c6d8043#96">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestGroupChatMultipleDeviceMergedNewUI:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3640399">test_group_chat_reactions</a>, id: 703202 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/294e251b89c641f18ede32f616c5dd6a?auth=7932d0eb0aec371854a0f4e6774263d5#185">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/3a7d6d2039084918b84ea8f36c33d1f6?auth=d6634ddf61edaf79e2b5cfdd825bb91f#110">Steps, video, logs</a></li></ul></p><p>Device 3:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/f1a669c2a3a641d08496c856520a602b?auth=f4c179cb570acbf2f83d790868473276#90">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3640374">test_group_chat_join_send_text_messages_push</a>, id: 702807 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/294e251b89c641f18ede32f616c5dd6a?auth=7932d0eb0aec371854a0f4e6774263d5#168">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/3a7d6d2039084918b84ea8f36c33d1f6?auth=d6634ddf61edaf79e2b5cfdd825bb91f#82">Steps, video, logs</a></li></ul></p><p>Device 3:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/f1a669c2a3a641d08496c856520a602b?auth=f4c179cb570acbf2f83d790868473276#82">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>3. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3640375">test_group_chat_offline_pn</a>, id: 702808 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/294e251b89c641f18ede32f616c5dd6a?auth=7932d0eb0aec371854a0f4e6774263d5#333">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/3a7d6d2039084918b84ea8f36c33d1f6?auth=d6634ddf61edaf79e2b5cfdd825bb91f#207">Steps, video, logs</a></li></ul></p><p>Device 3:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/f1a669c2a3a641d08496c856520a602b?auth=f4c179cb570acbf2f83d790868473276#205">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>4. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3640364">test_group_chat_pin_messages</a>, id: 702732 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/294e251b89c641f18ede32f616c5dd6a?auth=7932d0eb0aec371854a0f4e6774263d5#366">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/3a7d6d2039084918b84ea8f36c33d1f6?auth=d6634ddf61edaf79e2b5cfdd825bb91f#234">Steps, video, logs</a></li></ul></p><p>Device 3:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/f1a669c2a3a641d08496c856520a602b?auth=f4c179cb570acbf2f83d790868473276#232">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>5. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3640400">test_group_chat_send_image_save_and_share</a>, id: 703297 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/294e251b89c641f18ede32f616c5dd6a?auth=7932d0eb0aec371854a0f4e6774263d5#292">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/3a7d6d2039084918b84ea8f36c33d1f6?auth=d6634ddf61edaf79e2b5cfdd825bb91f#172">Steps, video, logs</a></li></ul></p><p>Device 3:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/f1a669c2a3a641d08496c856520a602b?auth=f4c179cb570acbf2f83d790868473276#145">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>6. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3640403">test_group_chat_mute_chat</a>, id: 703495 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/294e251b89c641f18ede32f616c5dd6a?auth=7932d0eb0aec371854a0f4e6774263d5#486">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/3a7d6d2039084918b84ea8f36c33d1f6?auth=d6634ddf61edaf79e2b5cfdd825bb91f#277">Steps, video, logs</a></li></ul></p><p>Device 3:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/f1a669c2a3a641d08496c856520a602b?auth=f4c179cb570acbf2f83d790868473276#246">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestCommunityMultipleDeviceMergedTwo:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3640384">test_community_leave</a>, id: 702845 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/5e4affab76de4b7a8a634bf99024378c?auth=97d5aa21f9544b0cf4a1029964a74857#304">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/d7f763514bcc4b24860091f23e40d351?auth=04e61cd99d120d0ca0617e5584cfe315#273">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3640373">test_community_mentions_push_notification</a>, id: 702786 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/5e4affab76de4b7a8a634bf99024378c?auth=97d5aa21f9544b0cf4a1029964a74857#139">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/d7f763514bcc4b24860091f23e40d351?auth=04e61cd99d120d0ca0617e5584cfe315#113">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>3. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3640376">test_community_markdown_support</a>, id: 702809 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/5e4affab76de4b7a8a634bf99024378c?auth=97d5aa21f9544b0cf4a1029964a74857#188">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/d7f763514bcc4b24860091f23e40d351?auth=04e61cd99d120d0ca0617e5584cfe315#171">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>4. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3640393">test_community_hashtag_links_to_community_channels</a>, id: 702948 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/5e4affab76de4b7a8a634bf99024378c?auth=97d5aa21f9544b0cf4a1029964a74857#307">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/d7f763514bcc4b24860091f23e40d351?auth=04e61cd99d120d0ca0617e5584cfe315#295">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>5. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3640406">test_community_join_when_node_owner_offline</a>, id: 703629 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/5e4affab76de4b7a8a634bf99024378c?auth=97d5aa21f9544b0cf4a1029964a74857#498">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/d7f763514bcc4b24860091f23e40d351?auth=04e61cd99d120d0ca0617e5584cfe315#476">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestActivityCenterContactRequestMultipleDevicePR:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3640387">test_activity_center_contact_request_accept_swipe_mark_all_as_read</a>, id: 702851 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/a09319a961834f8891dea39d22a77483?auth=2b2625d97e2ebdbdfe6c881eb48a0e36#103">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/596798caa9f9468fb11e9d949bb0626f?auth=4e96478e39578d4e2cf7d6ce9c29003e#116">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3640386">test_activity_center_contact_request_decline</a>, id: 702850 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/a09319a961834f8891dea39d22a77483?auth=2b2625d97e2ebdbdfe6c881eb48a0e36#60">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/596798caa9f9468fb11e9d949bb0626f?auth=4e96478e39578d4e2cf7d6ce9c29003e#54">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestActivityMultipleDevicePR:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3640392">test_activity_center_reply_read_unread_delete_filter_swipe</a>, id: 702947 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/501773a27382430f89e2fba70ea22a80?auth=07cfa9edd73f4bd9304124abadcdf239#127">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/1929df44f3e1495ebdbff83ee56223f8?auth=2a6723b4b380071029debcc15f12db1b#97">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table></details>

--- a/modules/react-native-status/android/src/main/java/im/status/ethereum/module/NetworkManager.kt
+++ b/modules/react-native-status/android/src/main/java/im/status/ethereum/module/NetworkManager.kt
@@ -36,11 +36,6 @@ class NetworkManager(private val reactContext: ReactApplicationContext) : ReactC
     @ReactMethod
     fun inputConnectionStringForBootstrapping(connectionString: String, configJSON: String, callback: Callback) {
         val jsonConfig = JSONObject(configJSON)
-        val receiverConfig = jsonConfig.getJSONObject("receiverConfig")
-        val keyStorePath = utils.pathCombine(utils.getNoBackupDirectory(), "/keystore")
-        receiverConfig.put("keystorePath", keyStorePath)
-        receiverConfig.getJSONObject("nodeConfig").put("rootDataDir", utils.getNoBackupDirectory())
-
         utils.executeRunnableStatusGoMethod(
             { Statusgo.inputConnectionStringForBootstrapping(connectionString, jsonConfig.toString()) },
             callback

--- a/modules/react-native-status/ios/RCTStatus/NetworkManager.m
+++ b/modules/react-native-status/ios/RCTStatus/NetworkManager.m
@@ -35,21 +35,7 @@ RCT_EXPORT_METHOD(inputConnectionStringForBootstrapping:(NSString *)cs
         configJSON:(NSString *)configJSON
         callback:(RCTResponseSenderBlock)callback) {
 
-    NSData *configData = [configJSON dataUsingEncoding:NSUTF8StringEncoding];
-    NSError *error;
-    NSMutableDictionary *configDict = [NSJSONSerialization JSONObjectWithData:configData options:NSJSONReadingMutableContainers error:&error];
-    NSMutableDictionary *receiverConfig = configDict[@"receiverConfig"];
-    NSMutableDictionary *nodeConfig = receiverConfig[@"nodeConfig"];
-    NSFileManager *fileManager = [NSFileManager defaultManager];
-    NSURL *rootUrl =[[fileManager URLsForDirectory:NSLibraryDirectory inDomains:NSUserDomainMask] lastObject];
-    NSURL *multiaccountKeystoreDir = [rootUrl URLByAppendingPathComponent:@"keystore"];
-    NSString *keystoreDir = multiaccountKeystoreDir.path;
-    NSString *rootDataDir = rootUrl.path;
-
-    [receiverConfig setValue:keystoreDir forKey:@"keystorePath"];
-    [nodeConfig setValue:rootDataDir forKey:@"rootDataDir"];
-    NSString *modifiedConfigJSON = [Utils jsonStringWithPrettyPrint:NO fromDictionary:configDict];
-    NSString *result = StatusgoInputConnectionStringForBootstrapping(cs, modifiedConfigJSON);
+    NSString *result = StatusgoInputConnectionStringForBootstrapping(cs, configJSON);
     callback(@[result]);
 }
 

--- a/src/status_im/config.cljs
+++ b/src/status_im/config.cljs
@@ -123,8 +123,6 @@
    :waku.test
    ["enrtree://AOGYWMBYOUIMOENHXCHILPKY3ZRFEULMFI4DOM442QSZ73TT2A7VI@test.waku.nodes.status.im"]})
 
-(def default-kdf-iterations 3200)
-
 (def community-accounts-selection-enabled? true)
 (def fetch-messages-enabled? (enabled? (get-config :FETCH_MESSAGES_ENABLED "1")))
 (def test-networks-enabled? (enabled? (get-config :TEST_NETWORKS_ENABLED "0")))

--- a/src/status_im/contexts/syncing/events.cljs
+++ b/src/status_im/contexts/syncing/events.cljs
@@ -52,8 +52,7 @@
   (let [config-map (.stringify js/JSON
                                (clj->js
                                 {:receiverConfig
-                                 {:createAccount (profile.config/create)
-                                  :deviceType    platform/os}}))]
+                                 {:createAccount (profile.config/create)}}))]
     (rf/dispatch [:syncing/update-role constants/local-pairing-role-receiver])
     (native-module/input-connection-string-for-bootstrapping
      connection-string

--- a/src/status_im/contexts/syncing/events.cljs
+++ b/src/status_im/contexts/syncing/events.cljs
@@ -3,8 +3,8 @@
     [clojure.string :as string]
     [native-module.core :as native-module]
     [react-native.platform :as platform]
-    [status-im.config :as config]
     [status-im.constants :as constants]
+    [status-im.contexts.profile.config :as profile.config]
     [status-im.contexts.syncing.utils :as sync-utils]
     [taoensso.timbre :as log]
     [utils.i18n :as i18n]
@@ -21,26 +21,6 @@
   {:events [:syncing/clear-states]}
   [{:keys [db]} role]
   {:db (dissoc db :syncing)})
-
-(defn- get-default-node-config
-  []
-  (let [log-level  config/log-level
-        log-config (if (empty? log-level)
-                     {:LogLevel "ERROR" :LogEnabled false}
-                     {:LogLevel log-level :LogEnabled true})]
-    (merge {:WalletConfig (cond-> {:Enabled true}
-                            (not= config/opensea-api-key "")
-                            (assoc :OpenseaAPIKey config/opensea-api-key))
-            :WakuV2Config {;; Temporary fix until https://github.com/status-im/status-go/issues/3024
-                           ;; is resolved
-                           :Nameserver  "8.8.8.8"
-                           :LightClient true}
-            :ShhextConfig {:VerifyTransactionURL     config/verify-transaction-url
-                           :VerifyENSURL             config/verify-ens-url
-                           :VerifyENSContractAddress config/verify-ens-contract-address
-                           :VerifyTransactionChainID config/verify-transaction-chain-id}}
-           log-config)))
-
 (defn- input-connection-string-callback
   [res]
   (log/info "[local-pairing] input-connection-string-for-bootstrapping callback"
@@ -69,24 +49,16 @@
 (rf/defn initiate-local-pairing-with-connection-string
   {:events [:syncing/input-connection-string-for-bootstrapping]}
   [{:keys [db]} connection-string]
-  (let [default-node-config-string (.stringify js/JSON (clj->js (get-default-node-config)))
-        callback
-        (fn [final-node-config]
-          (let [config-map (.stringify js/JSON
-                                       (clj->js
-                                        {:receiverConfig
-                                         {:kdfIterations config/default-kdf-iterations
-                                          :nodeConfig final-node-config
-                                          :settingCurrentNetwork config/default-network
-                                          :deviceType platform/os
-                                          :deviceName
-                                          (native-module/get-installation-name)}}))]
-            (rf/dispatch [:syncing/update-role constants/local-pairing-role-receiver])
-            (native-module/input-connection-string-for-bootstrapping
-             connection-string
-             config-map
-             input-connection-string-callback)))]
-    (native-module/prepare-dir-and-update-config "" default-node-config-string callback)))
+  (let [config-map (.stringify js/JSON
+                               (clj->js
+                                {:receiverConfig
+                                 {:createAccount (profile.config/create)
+                                  :deviceType    platform/os}}))]
+    (rf/dispatch [:syncing/update-role constants/local-pairing-role-receiver])
+    (native-module/input-connection-string-for-bootstrapping
+     connection-string
+     config-map
+     input-connection-string-callback)))
 
 (rf/defn preparations-for-connection-string
   {:events [:syncing/get-connection-string]}

--- a/status-go-version.json
+++ b/status-go-version.json
@@ -3,7 +3,7 @@
     "_comment": "Instead use: scripts/update-status-go.sh <rev>",
     "owner": "status-im",
     "repo": "status-go",
-    "version": "v0.183.43",
-    "commit-sha1": "1f7816dd1a5dbf8ecc4754fd90eea4431648ff69",
-    "src-sha256": "1qy4h9glx6qhjwrwjjqx0kl292p6psd3iv4zqf666pz3w0kr1nsn"
+    "version": "chore/pairing-config-create-account",
+    "commit-sha1": "01f7787c45060cb682150a6cb2c20a03f1d295ae",
+    "src-sha256": "0hddsry7lfz5xi8jgz56cj06pwy3flwdll70pc7wn2lc6zad8jxn"
 }

